lib/puppet_x/templates/invoke_ps_command.erb

```diff
@@ -23,7 +215,10 @@ public class PuppetPSHost : PSHost
     public override Guid InstanceId { get { return _hostId; } }
     public override string Name { get { return "PuppetPSHost"; } }
     public override Version Version { get { return new Version(1, 0); } }
```

Comment:
I like the idea of matching the PSHost Version to the module version but I can see @Iristyle point of view.  If it could be easily automated during the module build process then maybe...